edxval/api.py

```diff
@@ -316,27 +318,42 @@ def get_url_for_profile(edx_video_id, profile):
 
 def _get_videos_for_filter(
         video_filter,
-        sort_field=None,
-        sort_dir=SortDirection.asc
+        sort_field=VideoSortField.created,
+        sort_dir=SortDirection.desc,
```

Comment:
This would change the sorting as well as direction in case of no pagination. Can we make sure that we don't make such changes ?